MainModule/Server/Commands/Moderators.lua

```diff
@@ -2062,7 +2062,7 @@ return function(Vargs, env)
 						`Question: {question}`;
 						`Total Responses: {total}`;
 						`Didn't Vote: {#players-total}`;
-						`Time Left: {math.max(0, 120 - (os.clock()-startTime))}`;
```

Comment:
string.format is better for numerical strings because you can specify the decimal amount

Reply:
We don't need a decimal place, just the number